src/encoded/static/components/genome_browser.js

```diff
@@ -20,6 +21,55 @@ const GV_COORDINATES_KEY = 'ENCODE-GV-coordinates';
 const GV_COORDINATES_ASSEMBLY = 'ENCODE-GV-assembly';
 const GV_COORDINATES_ANNOTATION = 'ENCODE-GV-annotation';
 
+const colorChromatinState = {
+    'Active TSS': '#ff0000',
+    'Flanking TSS': '#ff4400',
+    'Flanking TSS upstream': '#ff4500',
+    'Flanking TSS downstream': '#ff4500',
+    'Strong transcription': '#008000',
+    'Weak transcription': '#006400',
+    'Genic enhancer 1': '#c4e105',
+    'Genic enhancer 2': '#c4e105',
+    'Active enhancer 1': '#ffc44d',
+    'Active enhancer 2': '#ffc44d',
+    'Weak enhancer': '#ffff00',
+    'ZNF genes & repeats': '#66cdaa',
+    Heterochromatin: '#8a91d0',
+    'Bivalent/Poised TSS': '#cd5c5c',
+    'Bivalent enhancer': '#bdb86b',
+    'Repressed PolyComb': '#808080, #8937df',
+    'Weak Repressed PolyComb': '#c0c0c0, #9750e3',
+    'Quiescent/Low': '#ffffff',
+};
+
+const colorCCREs = {
+    'Promoter-like': '#ff0000',
+    'Proximal enhancer-like': '#ffa700',
+    'Distal enhancer-like': '#ffcd00',
+    'DNase-H3K4me3': '#ffaaaa',
+    'CTCF-only': '#00b0f0',
+    'DNase-only': '#06da93',
+    'Low-DNase': '#ffffff',
+};
+
+const colorGenome = {
```

Comment:
Is there a reason why you use passive voice like "colorGenome" rather than active voice like "GenomeColors"?